build_external/bin/clean_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Basic setup, check we have a legal Distro / Version combo
+Distro=$1
+Version=$2
+Repo=$(cd $(dirname $0) && cd ../.. && pwd)
+Base=$Repo/build_external
+Arch=$(uname)
+
+TagName=clean_install_"$Distro"_"$Version":latest
+Dockerfile=$Base/clean_install_"$Distro"_"$Version".Dockerfile
+
+if [ ! -e "$Dockerfile" ]; then
+    echo "Can't run installer on $Distro/$Version - $Dockerfile not found"
+    exit -1
+fi
+
+if [ "$Arch" == "Linux" ]; then
+    sudo docker build -f "$Dockerfile" "$Repo" -t "$TagName"
```

Comment:
`sudo` is almost never necessary with Docker in general if you have Docker setup (_which it should be by default_) in such a way as you were a member of the `docker` group and the `/var/lib/docker.sock` UNIX socket has group read/write permissions.